src/index.ts

```diff
@@ -333,6 +333,18 @@ function main() {
                 })
                 .then((ans: Answers) => {
                     if (program.sku.toLowerCase() === solutionSkus[solutionSkus.local]) {
+                        const dateTime = Date.now().toString();
```

Comment:
Since local deployment will also create AAD application, service principal and credentials, there is not so much difference comparing to basic and standard deployment.

Reply:
Hi, I separated out the code as I believe adminPassword, sshFilePath is required only for VM of the basic deployment. I may be wrong here. Could you clarify?